### PR TITLE
New masks

### PR DESCRIPTION
**Pull request by NeonNik2245**
## About The Pull Request
Adds 3 new masks:
* Kitsune mask
* Facescarf
* Rebellion mask
All masks (and facescarf) are recolorable. And also thanks to Kovac for the scarf sprites
## Why It's Good For The Game
Cool masks for make character style more unique. All are recolourable.

## Changelog
:cl:
add: 3 new masks for your characters (thanks to Kovac for the scarf sprites)
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
Hi, since this is the second time this PR has been put up, I was wondering if you could something. Could you possibly provide some sources for these sprites? Where did they come from?

**Comment by NeonNik2245** [comment]
> Hi, since this is the second time this PR has been put up, I was wondering if you could something. Could you possibly provide some sources for these sprites? Where did they come from?

Yes, it's the second time. I borrow kitsune mask and Facescarf from BlueMoon build (https://github.com/BlueMoon-Labs/MOLOT-BlueMoon-Station). Rebellion mask is made all by myself (You can guess by her appearance :D). But I also changed Facescarf (new lowered variant, greyscale) and kitsune mask (resprite it very much and add greyscale)

**Comment by NeonNik2245** [comment]
> Make the neck scarf colorable please, masks look okay but I'm not the fan of the far most right one

It's already

**Comment by NeonNik2245** [comment]
> Hi, since this is the second time this PR has been put up, I was wondering if you could do something. Could you possibly provide some sources for these sprites? Where did they come from?

image
There is original of the mask

**Review by Wallemations — CHANGES_REQUESTED** [review]
Fix your changelog

**Comment by LemonInTheDark** [comment]
Please credit the people whose work you're pring here

**Comment by NeonNik2245** [comment]
> Fix your changelog

Done!

**Comment by Wallemations** [comment]
> Please credit the people whose work you're pring here (if possible)

I second this, be sure to add that credit to your changelog as well

**Comment by Zargoar** [comment]
Great first PR!

**Comment by NeonNik2245** [comment]
> > Please credit the people whose work you're pring here (if possible)
> 
> I second this, be sure to add that credit to your changelog as well

I think it's done too for now?

**Review by Wallemations — APPROVED** [review]

**Review by dragomagol — CHANGES_REQUESTED** [review]
Looks mostly good, just a few style and grammar improvements

File: code/modules/clothing/masks/bandana.dm [hunk 1]
```diff
@@ -200,3 +200,35 @@
 	desc = "A fine black bandana with nanotech lining and a skull emblem."
 	greyscale_colors = "#2e2e2e#C6C6C6"
 	flags_1 = NONE
+
+/obj/item/clothing/mask/facescarf
```
[the remark this answers is not in the document]
Comment (NeonNik2245) [reply code/modules/clothing/masks/bandana.dm:204]: Yes, that makes sense, I'll make it subtype

**Comment by NeonNik2245** [comment]
> Looks mostly good, just a few style and grammar improvements

And all done!

File: code/modules/clothing/masks/bandana.dm [hunk 2]
```diff
@@ -200,3 +200,35 @@
...
+	flags_cover = MASKCOVERSMOUTH
+	visor_flags_cover = MASKCOVERSMOUTH
+	custom_price = PAYCHECK_CREW
+	tint = 0
+	greyscale_colors = "#eeeeee"
+	greyscale_config = /datum/greyscale_config/facescarf
+	greyscale_config_worn = /datum/greyscale_config/facescarf/worn
+	greyscale_config_inhand_left = /datum/greyscale_config/facescarf/inhands_left
+	greyscale_config_inhand_right = /datum/greyscale_config/facescarf/inhands_right
+	flags_1 = IS_PLAYER_COLORABLE_1
```
Comment (dragomagol) [inline code/modules/clothing/masks/bandana.dm:225]:
```suggestion
/obj/item/clothing/mask/bandana/facescarf
	name = "facescarf"
	desc = "Cover your face like in the cowboy movies. It also has a breathtube so you can wear it everywhere!"
	actions_types = /datum/action/item_action/adjust
	icon_state = "facescarf"
	inhand_icon_state = "greyscale_facescarf"
	alternate_worn_layer = BACK_LAYER
	clothing_flags = BLOCK_GAS_SMOKE_EFFECT|MASKINTERNALS
	w_class = WEIGHT_CLASS_SMALL
	visor_flags = BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
	custom_price = PAYCHECK_CREW
	greyscale_colors = "#eeeeee"
	greyscale_config = /datum/greyscale_config/facescarf
	greyscale_config_worn = /datum/greyscale_config/facescarf/worn
	greyscale_config_inhand_left = /datum/greyscale_config/facescarf/inhands_left
	greyscale_config_inhand_right = /datum/greyscale_config/facescarf/inhands_right
```

All you need to do is remove the shared variables (and tint, which should already be 0). I would also check to see if the AltClick and examine procs still work the way you expect them to in-game

**Comment by NeonNik2245** [comment]
> Looks mostly good, just a few style and grammar improvements

There is problem with subtype of bandana: bandana has `/obj/item/clothing/mask/bandana/AltClick(mob/user)`, that break facescarf because of another work of that action. Better leave facescarf be it's own mask type

**Comment by NeonNik2245** [comment]
> > Looks mostly good, just a few style and grammar improvements
> 
> There is problem with subtype of bandana: bandana has `/obj/item/clothing/mask/bandana/AltClick(mob/user)`, that break facescarf because of another work of that action. Better leave facescarf be it's own mask type

image
That works on old version

**Pull request merged**